app/config.py

```diff
@@ -44,51 +42,32 @@
 CONFIG = configparser.ConfigParser()
 CONFIG.read(INI_FILE)
 
-# Web3設定
-WEB3_HTTP_PROVIDER = os.environ.get("WEB3_HTTP_PROVIDER") or 'http://localhost:8545'
-WEB3_HTTP_PROVIDER_STANDBY = [node.strip() for node in os.environ.get("WEB3_HTTP_PROVIDER_STANDBY").split(",")] \
-    if os.environ.get("WEB3_HTTP_PROVIDER_STANDBY") else []
-WEB3_CHAINID = os.environ.get("WEB3_CHAINID") or CONFIG['web3']['chainid']
-# トランザクションレシート待機時間（秒）
-TRANSACTION_WAIT_TIMEOUT = int(os.environ.get("TRANSACTION_WAIT_TIMEOUT")) \
-    if os.environ.get("TRANSACTION_WAIT_TIMEOUT") else 5
-TRANSACTION_WAIT_POLL_LATENCY = float(os.environ.get("TRANSACTION_WAIT_POLL_LATENCY")) \
-    if os.environ.get("TRANSACTION_WAIT_POLL_LATENCY") else 0.5
+# Issuing companies list
+COMPANY_LIST_URL = os.environ.get("COMPANY_LIST_URL")
+COMPANY_LIST_LOCAL_MODE = True if os.environ.get("COMPANY_LIST_LOCAL_MODE") == "1" else False
+COMPANY_LIST_SLEEP_INTERVAL = int(os.environ.get("COMPANY_LIST_SLEEP_INTERVAL")) if os.environ.get("COMPANY_LIST_SLEEP_INTERVAL") else 3600
 
-# サーバ設定
+####################################################
+# Server settings
+####################################################
+# Worker thread 
 WORKER_COUNT = int(os.environ.get("WORKER_COUNT")) if os.environ.get("WORKER_COUNT") else 8
+
+# HTTP request timeout
 REQUEST_TIMEOUT = (3.0, 7.5)
-SLEEP_INTERVAL = int(os.environ.get("SLEEP_INTERVAL")) if os.environ.get("SLEEP_INTERVAL") else 3
 
-# プロセッサ設定
-# ブロック同期状態監視間隔 (秒)
-BLOCK_SYNC_STATUS_SLEEP_INTERVAL = int(os.environ.get("BLOCK_SYNC_STATUS_SLEEP_INTERVAL", SLEEP_INTERVAL))
-# ブロック同期状態の判定に使用する監視データの数
-BLOCK_SYNC_STATUS_CALC_PERIOD = int(os.environ.get("BLOCK_SYNC_STATUS_CALC_PERIOD", 5))
-# ブロック同期残のしきい値（block）
-# NOTE: highestBlock と currentBlock の差のしきい値
-BLOCK_SYNC_REMAINING_THRESHOLD = int(os.environ.get("BLOCK_SYNC_REMAINING_THRESHOLD", 2))
-# ブロック同期停止と判断するブロック生成速度のしきい値 (%)
-# NOTE: Quorum Validator 4台中1台がクラッシュ障害状態のとき、ブロック生成速度は 20% 〜 35% ぐらいになる
-if APP_ENV == "local":
-    BLOCK_GENERATION_SPEED_THRESHOLD = int(os.environ.get("BLOCK_GENERATION_SPEED_THRESHOLD")) \
-        if os.environ.get("BLOCK_GENERATION_SPEED_THRESHOLD") else 0
-else:
-    BLOCK_GENERATION_SPEED_THRESHOLD = int(os.environ.get("BLOCK_GENERATION_SPEED_THRESHOLD")) \
-        if os.environ.get("BLOCK_GENERATION_SPEED_THRESHOLD") else 10
-WEB3_REQUEST_RETRY_COUNT = int(os.environ.get("WEB3_REQUEST_RETRY_COUNT")) if os.environ.get(
-    "WEB3_REQUEST_RETRY_COUNT") else 3
-WEB3_REQUEST_WAIT_TIME = int(os.environ.get("WEB3_REQUEST_WAIT_TIME")) \
-    if os.environ.get("WEB3_REQUEST_WAIT_TIME") else BLOCK_SYNC_STATUS_SLEEP_INTERVAL  # Same batch interval
+# Batch processing interval
+BATCH_PROCESS_INTERVAL = int(os.environ.get("SLEEP_INTERVAL")) if os.environ.get("SLEEP_INTERVAL") else 3
```

Comment:
I think below line would be better.
(In fact, `BATCH_PROCESS_INTERVAL` is not used and has no effect.)
```
BATCH_PROCESS_INTERVAL = int(os.environ.get("BATCH_PROCESS_INTERVAL")) if os.environ.get("BATCH_PROCESS_INTERVAL") else 3
```

awesome PR